search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/filter/UserTaskFilterTransformer.java

```diff
@@ -119,16 +110,20 @@ private SearchQuery getUserTaskKeysQuery(final List<Long> userTaskKeys) {
     return longTerms("key", userTaskKeys);
   }
 
-  private SearchQuery getCandidateUsersQuery(final List<String> candidateUsers) {
-    return stringTerms("candidateUsers", candidateUsers);
+  private List<SearchQuery> getCandidateUsersQuery(final List<Operation<String>> candidateUsers) {
+    return stringOperations("candidateUsers", candidateUsers);
+  }
+
+  private List<SearchQuery> getCandidateGroupsQuery(final List<Operation<String>> candidateGroups) {
+    return stringOperations("candidateGroups", candidateGroups);
   }
 
-  private SearchQuery getCandidateGroupsQuery(final List<String> candidateGroups) {
-    return stringTerms("candidateGroups", candidateGroups);
+  private List<SearchQuery> getAssigneesQuery(final List<Operation<String>> assignees) {
+    return stringOperations("assignee", assignees);
   }
 
-  private SearchQuery getAssigneesQuery(final List<String> assignee) {
-    return stringTerms("assignee", assignee);
+  private List<SearchQuery> getPrioritiesQuery(final List<Operation<Integer>> priorities) {
+    return intOperations("priority", priorities);
```

Comment:
❓ This is a new filter option that is untested, right? The other fields are already covered by existing tests and don't change so the existing tests should verify that we don't introduce any regressions with this, right?

Reply:
Yes, it should be tested too. I will add tests.